src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsComputeQuota.ps1

```diff
@@ -31,6 +31,12 @@ Changes may cause incorrect behavior and will be lost if the code is regenerated
 .PARAMETER VirtualMachineCount
     Maximum number of virtual machines allowed.
 
+.PARAMETER MaxAllocationStandardManagedDisksAndSnapshots
```

Comment:
Same as above.

Reply:
This has been discussed and PM has opened a bug to track this and get done.